README-quick-start.md

```diff
@@ -0,0 +1,70 @@
+# How to install CMS locally from scratch
+This document cherry picks some commands that can already be found in [README-short.md](https://github.com/office-for-students/wagtail-CMS/blob/develop/README-short.md)
+
+Notes:
+1. Python 3.6.8 is referenced in [Confluence](https://mobilisecloud.atlassian.net/wiki/spaces/OFS/pages/236912643/Managing+multiple+Python+versions) and [Docker](https://github.com/office-for-students/wagtail-CMS/blob/master/Dockerfile). If you don't already have 3.6 installed, a later version should work too.
+
+
+```
+# First, delete any existing Docker containers/images using CLI or UI, then...
+
+# Create a fresh directory to work in 
+mkdir -p ~/temp/demo-wagtail-fresh-install 
+cd ~/temp/demo-wagtail-fresh-install
+
+# Clone the relevant directories
+git clone https://github.com/office-for-students/wagtail-CMS
+git clone https://gitlab.mgmt.mobilise.cloud/mobilise-dev/ofs-secrets
```

Comment:
as this is a public repo, should that be listed here

Reply:
`ofs-secrets` is not public